week-03/assessment/solar-farm/solar-farm/src/main/java/learn/solar/data/DataException.java

```diff
@@ -0,0 +1,14 @@
+package learn.solar.data;
+
+public class DataException extends Exception{
+    public DataException(String message){
+        super(message);
+    }
+    public DataException(String message, Throwable cause){
+        super(message, cause);
+    }
+}
+    //Custom data layer exception.
+
+        //public DataException(String, Throwable)` -- constructor,
+        //Throwable arg is the root cause exception
```

Comment:
I love seeing these notes! They become a valuable resource when going back and looking at your assessment in the future